client/apps/edit/components/admin/index.coffee

```diff
@@ -48,28 +48,77 @@ module.exports = React.createClass
-        printTitle section: 'Article', className: 'article', onClick: @setActiveSection
+      section {
+        className: 'edit-admin--article' + @getActiveSection 'article'
+      },
+        printTitle {
+          section: 'Article'
+          className: 'article'
+          onClick: @setActiveSection
+        }
         if @isActiveSection 'article'
-          Article {article: @props.article, channel: @props.channel, onChange: @onChange}
+          Article {
+            article: @props.article
+            channel: @props.channel
+            onChange: @onChange
+        }
 
-      section { className: 'edit-admin--featuring' + @getActiveSection 'featuring' },
-        printTitle section:  'Featuring', className: 'featuring', onClick: @setActiveSection
+      section {
+        className: 'edit-admin--featuring' + @getActiveSection 'featuring'
+      },
+        printTitle {
+          section:  'Featuring'
+          className: 'featuring'
+          onClick: @setActiveSection
+        }
         if @isActiveSection 'featuring'
-          Featuring {article: @props.article, channel: @props.channel, onChange: @onChange}
+          Featuring {
+            article: @props.article
+            channel: @props.channel
+            onChange: @onChange
+          }
 
       if @props.channel.hasFeature 'superArticle'
-        section { className: 'edit-admin--super-article' + @getActiveSection 'super-article' },
-          printTitle section: 'Super Article', className: 'super-article', onClick: @setActiveSection
+        section {
+          className: 'edit-admin--super-article' + @getActiveSection 'super-article'
+        },
+          printTitle {
+            section: 'Super Article'
+            className: 'super-article'
+            onClick: @setActiveSection
+          }
           if @isActiveSection 'super-article'
-            SuperArticle {article: @props.article, channel: @props.channel, onChange: @onChange}
+            SuperArticle {
+              article: @props.article
+              channel: @props.channel
+              onChange: @onChange
+            }
 
-      section { className: 'edit-admin--additional-appearances' + @getActiveSection 'additional-appearances' },
-        printTitle section: 'Additional Appearances', className: 'additional-appearances', onClick: @setActiveSection
+      section {
+        className: 'edit-admin--additional-appearances' + @getActiveSection 'additional-appearances'
+      },
+        printTitle {
+          section: 'Additional Appearances'
+          className: 'additional-appearances'
+          onClick: @setActiveSection
+        }
         if @isActiveSection 'additional-appearances'
-          Appearances {article: @props.article, channel: @props.channel, onChange: @onChange}
\ No newline at end of file
+          Appearances {
+            article: @props.article
+            channel: @props.channel
+            onChange: @onChange
+          }
```

Comment:
This file is irrelevant to the PR, but I just noticed most of the lines here were longer than 100 chars and generally it's a bit easier to read when they are broken out to <100.

In Sublime, there's an option called Ruler which is nice to turn on as a guide (note the faint vertical line):
![image](https://cloud.githubusercontent.com/assets/2236794/26328562/8eeaff7a-3f11-11e7-8f85-8f107899774f.png)